code/config/resCodes/user/donation.js

```diff
@@ -0,0 +1,28 @@
+resCode = {
+  report:{
```

Comment:
This should be named after the function. So "saveDonation" instead of "report"

Reply:
Fair enough, my bad for not seeing that one. I've gone through and changed it in all areas it was used.